tests/steamship_tests/app/integration/test_package_instance.py

```diff
@@ -157,6 +157,22 @@ def replace_string(s: str):
             steamship_dir_fixed = json.loads(replace_string(json.dumps(steamship_dir)))
             assert steamship_dir_fixed == steamship_dir_golden
 
+        # Test that we can call a task which SCHEDULES another task
+        future_greet_resp = instance.invoke("future_greet", name="Unicorn")
+        future_task = Task(client=client, **future_greet_resp)
+        future_task.wait()
+        future_task_bytes = base64.b64decode(future_task.output)
```

Comment:
Is there a way for the task output not to be b64encoded?

Reply:
Yeah this is such a tricky spot -- base64 is super annoying, but if we don't do it, then we either need to: (1) forbid base64 output or (2) base64 sometimes but not always.

All of those options feel not great to me -- should we pursue some sort of "test the base64 decode and do it automatically if it's viable" trick? 

That way the user always gets back a reasonable python type